### PR TITLE
Fix to compile for upstream changes

### PR DESCRIPTION
**Pull request by RabbitWhite1**

### CODE REVIEW

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ jobs:
 
     - name: Pull P4C and link P4RuntimeSmith.
       run: |
-        git clone --recurse-submodules -j2 https://github.com/p4lang/p4c/ $GITHUB_WORKSPACE/../p4c
+        git clone --recurse-submodules -b fruffy/flay -j2 https://github.com/fruffy/p4c/ $GITHUB_WORKSPACE/../p4c
```
Comment (fruffy) [inline .github/workflows/ci-test.yml:68]: Line 97 needs to change from RELEASE to Release

**Review by fruffy — APPROVED** [review]

**Comment by fruffy** [comment]
I am not sure whether we need the Flay branch really for RtSmith.

**Comment by fruffy** [comment]
Merging for now, might revert at some later point.

**Comment by RabbitWhite1** [comment]
> I am not sure whether we need the Flay branch really for RtSmith.

Ah I just change it mostly for ease of developing RtSmith and Flay at the same time. We may always revert when necessary.

**Pull request merged**